icu4c/source/common/unicode/chariter.h

```diff
@@ -142,7 +142,7 @@ class U_COMMON_API ForwardCharacterIterator : public UObject {
      * @return a UClassID for this ForwardCharacterIterator 
      * @stable ICU 2.0
      */
-    virtual UClassID getDynamicClassID(void) const = 0;
+    //virtual UClassID getDynamicClassID(void) const = 0;
```

Comment:
Same question about this being commented out.

Reply:
virtual UClassID getDynamicClassID(void) const override = 0;